src/lib/clubPitchAnalytics.ts

```
@@ -0,0 +1,140 @@
+  const secNum = Math.max(0, Math.floor(seconds));
+  const mins = Math.floor(secNum / 60);
+  const secs = secNum % 60;
+  return `${mins}:${secs < 10 ? "0" : ""}${secs}`;
+}
+
+/**
+ * Aggregates 5-second interval retention curve analytics from raw audio telemetry pings (#4271).
+ */
+export function calculatePitchRetentionCurve(
+  rawPings: PitchTelemetryPing[],
+  pitchDurationSec: number = 60
```

ESLint

[error] Insert `,`

(prettier/prettier)